src/bert/utils.py

```diff
@@ -152,3 +152,13 @@ def get_output_dim(self):
             return 2 * self.model.config.hidden_size
         else:
             return self.model.config.hidden_size
+
+    def hatch_LM_head(self, args):
```

Comment:
What does `hatch` mean? Could you make the docstring a little clearer as to what this is doing?

Reply:
Well, this name is not very intuitive. I have changed it and added notes